test_harness/include/hebench_types_harness.h

```diff
@@ -33,6 +33,9 @@ using unique_ptr_custom_deleter = std::unique_ptr<T, std::function<void(T *)>>;
 constexpr const char *FileNameNoExtReport  = "report";
 constexpr const char *FileNameNoExtSummary = "summary";
 
+constexpr char separator = '/';
```

Comment:
I believe this works because the "portable" separator of C++ `filesystem` is `/`, but I'm not sure. We'll have to keep it in mind when we get around to add Windows compatibility to make sure the separator is actually "portable."

For now, it looks good to me.